src/frontend/screens/Observation/index.tsx

```diff
@@ -86,15 +97,18 @@ export const ObservationScreen: NativeNavigationComponent<'Observation'> = ({
               <Text style={styles.textNotes}>{observation.tags.notes}</Text>
             </View>
           ) : null}
-
-          {/* {!!photos.length && (
+          {base64Uris.length > 0 && (
             <ThumbnailScrollView
-              photos={
-                photos
-              }
-              onPressPhoto={() => {}}
+              photos={base64Uris.map(({data}) => {
+                return !data
+                  ? undefined
```

Comment:
When does this happen?

Reply:
Whenever a query isn't resolved yet (image is loading)